.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,25 +9,30 @@ are the most critical to review. -->
 
 ### Author Checklist
 
-*All items are required. Please add a note to the item if the item is not applicable and
-please add links to any relevant follow up issues.*
+_All items are required. Please add a note to the item if the item is not applicable and
```

Comment:
i'm not sure why my editor changed these, but it doesn't impact anything... just changes markdown syntax